client/src/App.css

```diff
@@ -151,6 +155,32 @@ table, th, td {
 .unknown{
     background-color: #A8CAFF;
 }
+
+.link-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #596275;
+  left: 321px;
+  top: 1114px;
+  border: 1px solid #ccc;
+  box-sizing: border-box;
+  width: fit-content;
+  font-weight: 700;
+  padding: 0 1rem;
+  font-size: 1.25rem;
```

Comment:
Margine & padding better to be in precentage measure for better responsive web app on various devices